zebra-rpc/src/methods/get_block_template_rpcs/get_block_template.rs

```diff
@@ -131,10 +130,12 @@ where
     let block_verifier_router_response = block_verifier_router
         .ready()
         .await
-        .map_err(|error| Error {
-            code: ErrorCode::ServerError(0),
-            message: error.to_string(),
-            data: None,
+        .map_err(|error| {
+            ErrorObject::owned(
+                ErrorCode::ServerError(0).code(),
```

Comment:
Nitpick/Optional: `ErrorCode::ServerError(0).code()` could be replaced with `0` here, on line 228, and on line 264, but this file will need to be formatted again.